### ISSUE TITLE
pep8 import unused error

### ISSUE THREAD

**Issue by RazerM** [dreadatour/Flake8Lint#27]
On save in ST3, the following error is reported in the console:

```
An exception occured when running pep8 checker: 'int' object has no attribute 'lineno'
```
